sockeye/data_io.py

```diff
@@ -212,13 +212,11 @@ def define_bucket_batch_sizes(buckets: List[Tuple[int, int]],
             # Max number of sequences without exceeding batch size
             batch_size_seq = batch_size // padded_seq_len
             # Round down to closest multiple
-            batch_size_seq = (batch_size_seq // batch_sentences_multiple_of) * batch_sentences_multiple_of
+            batch_size_seq = (batch_size_seq // min_batch_step) * min_batch_step
```

Comment:
These lines were never updated to match the intended batching behavior.  They should have also been revised in the original PR.